app/public/dist/client/locales/en/translation.json

```diff
@@ -1256,383 +1256,383 @@
+    "CHARGE_BEAM": "Fire a concentrated bundle of electricity that propagates to a maximum of 3 adjacent enemy Pokémon, dealing [15,30,60,SP] SPECIAL to each.",
+    "POPULATION_BOMB": "Deal 10 SPECIAL [4,8,12,16,SP] times to the target.",
+    "SCREECH": "Reduce DEF of all adjacent enemy Pokémon by [1,2,4,SP].",
+    "SAND_TOMB": "Deal [10,20,40,SP] SPECIAL to the target and inflict them with SILENCE and PARALYSIS for [3,5,8] seconds.",
+    "WHIRLWIND": "Deal [40,80,120,SP] SPECIAL to the target and make them fly away.",
+    "EMPTY_LIGHT": "Spit poison directly at an enemy Pokémon, dealing [33,SP] SPECIAL and reducing their SPE_DEF by 3. The ability then rebounds to another enemy Pokémon with a lower SPE_DEF. (5 bounces maximum)",
+    "UNBOUND": "Unbinds itself to become Hoopa's true form and gains a humongous amount of power.",
+    "HYPERSPACE_FURY": "Using its many arms, unleashes a barrage of [4,SP] attacks. Each attack deals 15 SPECIAL to the target and lowers their DEF by 1.",
+    "SNIPE_SHOT": "Fire a shot of water from the user's index fingertips at the farthest enemy Pokémon, dealing [50,100,200,SP] SPECIAL to all enemy Pokémon in its path.",
+    "AIR_SLASH": "Deal [25,50,100,SP] SPECIAL to the target and inflict them with FLINCH for 7 seconds.",
+    "EGGSPLOSION": "Launch an explosive egg at the target, dealing [40,80,140,SP] SPECIAL to all adjacent enemy Pokémon and inflicting them with ARMOR_REDUCTION for 4 seconds. For each enemy Pokémon KO'd, there is a [30,LK]% chance of an egg appearing on the player's bench.",
+    "FLORAL_HEALING": "Heal HP equal to max PP of Comfey's holder or Comfey itself if not held.",
+    "VINE_WHIP": "Deal [100,SP] SPECIAL to the target and cause them and all adjacent enemy Pokémon to FLINCH for 3 seconds.",
+    "BODY_SLAM": "Deal [30,SP]% of the user's max HP as SPECIAL to the target.",
+    "BARB_BARRAGE": "Dive into the middle of the enemy team, dealing [15,30,60,SP] SPECIAL to all adjacent enemy Pokémon and inflict them with POISONNED for 3 seconds.",
+    "MAGIC_POWDER": "Gain [10,20,40,SP] SHIELD and SILENCE adjacent enemy Pokémon for [2,3,4] seconds.",
+    "INFERNAL_PARADE": "Shoot a returning Flame Wheel that has a [50,LK]% chance to apply BURN for 3s and dealing [30, SP] SPECIAL to all enemy Pokémon in its path. May hit twice.",
+    "RETALIATE": "Deal [15,30,60,SP] SPECIAL to the target +[10,15,25,SP] SPECIAL for each fallen allied Pokémon.",
+    "SLASH": "Deal [10,20,40,SP] SPECIAL to the target with a [30,60,90]% increased CRIT_CHANCE ratio. Can crit by default.",
+    "OUTRAGE": "Deal [100,150,200,SP]% ATK as SPECIAL to the target and adjacent enemy Pokémon and inflict the user with CONFUSION for 2 seconds.",
+    "LUNGE": "Jump to the enemy Pokémon with the highest ATK, dealing [50,SP] SPECIAL and reducing its ATK by [5,SP].",
+    "KNOCK_OFF": "Deal [90,SP] SPECIAL to the target, +[30,SP] SPECIAL for each item held by the target and they are removed.",
```

Comment:
i prefer the original version, not clear what 'they' refer to

Reply:
Fixed clarity, thanks